vcx/wrappers/java/vcx/local.properties

```diff
@@ -5,5 +5,5 @@
 # For customization when using a Version Control System, please read the
 # header note.
 #Mon Jun 25 13:30:41 IST 2018
-ndk.dir=/Users/evernym/Library/Android/sdk/ndk-bundle
-sdk.dir=/Users/evernym/Library/Android/sdk
+# ndk.dir=/Users/evernym/Library/Android/sdk/ndk-bundle
+# sdk.dir=/Users/evernym/Library/Android/sdk
```

Comment:
Naga, does this need to be uncommented so that the build of the .aar file will be successful?

Reply:
this is breaking the CI/CD. `[Android] NDK is missing a "platforms" directory.
[Android] If you are using NDK, verify the ndk.dir is set to a valid NDK directory.  It is currently set to /Users/evernym/Library/Android/sdk/ndk-bundle.
[Android] If you are not using NDK, unset the NDK variable from ANDROID_NDK_HOME or local.properties to remove this warning.` I think this file should not be checked in and must be used at user level

Reply:
I agree that the local.properties file should not be checked in and it should be added to the .gitignore file

Reply:
I agree that the local.properties file should be removed from the repo and added to the .gitignore file.